### DISCUSSION TITLE
It takes long time to discover device after the esp32 chip is plugged in and no mqtt-related information is seen in logs.

### DISCUSSION THREAD

**Discussion by racinmat** [fsaris/EspHome-AwoX-BLE-mesh-hub#100] [Q&A]
If I disconnect the esp32 chip and reconnect it, it takes over 10 minutes for the devices to be seen as available in HomeAssistant, and I don't see anything related to mqtt in the logs, even with the highest verbosity enabled, can I fix this issue somehow? 
All information including logs output and yaml is in #99

**Comment by fsaris** [comment]
What do you see in de logs of the esphome device?
